resources/js/pages/bookshelf/add.vue

```diff
@@ -11,9 +11,10 @@
       <p class="text-lg">
         {{ $t('welcomeAdd-paragraph') }}
       </p>
-      <BookshelfAddStep1 :disabled="currentStep !== 1 || !currentBookshelf" @select="selectBook" />
+      <BookshelfAddStep1 :disabled="currentStep !== 1 || !currentBookshelf" @select="selectBook" ref="bookSelection" />
       <BookshelfAddStep2 v-if="currentStep >= 2" :disabled="currentStep !== 2" @select="selectCondition" />
       <Button v-if="currentStep === 3" class="float-right mt-6" @click="addToBookshelf">Add to your Bookshelf</Button>
+      <Button v-if="selectedBook" theme="secondary" color="secondary" class="float-right mt-6" :class="currentStep == 2 ? 'mr-0' : 'mr-6'" @click="resetSelection">Start again</Button>
```

Comment:
Instead than add the logic of current step to define the margin right. You could just ALWAYS set Margin left on the Add to your Bookshelf Button!

- [ ] Remove the logic in the Reset button for MR
- [ ] Add the Margin Left on the Add to your Bookshelf button

Reply:
I can't really guess what I was thinking when I coded this. My bad. Made these changes and amended them.